File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>

int main(void)

{
    // Declare change
    float change;
    // Declare number of coins
    int count = 0;
    
    // Get change amount
    do
    {
        change = get_float("Change owed: \n");
    }
    while (change < 0);
 
    // Change from dollars to cents
    change = change * 100;
    // Change from float to int
    change = (int)(change + 0.5);
    
    // Quarters
    while (change >= 25)
    {
        change -= 25;
        count++;
    };

    // Dimes
    while (change >= 10)
    {
        change -= 10;
        count++;
    };

    // Nickles
    while (change >= 5)
    {
        change -= 5;
        count++;
    };

    // Pennies
    while (change > 0)
    {
        change -= 1;
        count++;
    };

    printf("%i\n", count);
}

File: pset6/cash/cash.py
from cs50 import get_float

change = 0
count = 0

# get change
while change <= 0:
    change = get_float("Change owed: ")
    
# make into cents
change = change * 100

# make into int
change = int(change)

# quarters
while change >= 25:
    change -= 25
    count += 1
    
# dimes
while change >= 10:
    change -= 10
    count += 1

# nickles    
while change >= 5:
    change -= 5
    count += 1
    
# pennies
while change > 0:
    change -= 1
    count += 1
 
# return number of coins  
print(count)